### ISSUE TITLE
NVDA & JAWS no longer read sub-menu entries in Chrome 93.0.4556.0+

### ISSUE THREAD

**Issue by westonkd** [tinymce/tinymce#7300]
## What is the current behavior? Describe the bug
When using arrow keys to navigate sub-menus in the TinyMCE menubar, NVDA + JAWS no longer read sub-menu items.

Example:
![menu-demo](https://user-images.githubusercontent.com/6128215/137370627-41da5c8b-0b53-4028-8cf4-697d975ea520.gif)
In the above example, NVDA and JAWS do not read the sub-menu items ("New document", "Preview", etc.) in Chrome  93.0.4556.0 and up

## Steps to reproduce
1. Navigate to the [TinyMCE full-featured demo](https://www.tiny.cloud/docs/demo/full-featured/) (or any TinyMCE instance with a menu bar) in Chrome 93.0.4556.0 or greater
2. Turn on NVDA or JAWS
3. Use the arrow keys to navigate the TinyMCE sub-menus (the "File" sub-menu, for example)
4. Notice that the screen reader fails to read the sub-menu items

## Suggested Fix
Currently, TinyMCE adds an `aria-owns` attribute to the `button` element that opens the sub-menu ("File" button in the above example). This `aria-owns` attribute points to the div containing the sub-menu.

Removing the `aria-owns` attribute from the button elements solves the problem in my testing. I'm assuming `aria-owns` was added to the button because the sub-menu is not nested hierarchically in markup with the button.

W3C ARIA examples and best practices ([here](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/menu-button/menu-button-1/menu-button-1.html), for example) do not recommend using `aria-owns` on a button.

I believe doing so is communicating to assistive technologies that the sub-menu div is nested _inside_ the button, which seems to be causing the issue.

## What is the expected behavior?
Screen readers should read sub-menu items in Chrome

## Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?
I've tested this issue in TinyMCE 5.9.2 and 5.8.1, it was reproducible in both.  Chrome must be 93.0.4556.0 or greater to reproduce the issue. Tested with latest NVDA and JAWS

Thanks!
